src/css.js

```diff
@@ -113,6 +113,13 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		val = Math.round( elem.getBoundingClientRect()[ name ] * 100 );
```

Comment:
One other thing: as you can see in [src/offset.js#L93-L98](https://github.com/jquery/jquery/blob/842958e7aecd0d75a7ee9e2aaec83457701aa2f3/src/offset.js#L93-L98), in IE (all versions) running `getBoundingClientRect()` on disconnected nodes throws an error and we support `elem.css('width')` on disconnected elements. So you need to set the value only for non-disconnected node, you can use a similar `getClientRects()` check as in the offset module.

I guess we could abstract this function and put in a separate module but this optimization may as well be done later, for now we just need to ensure this won't throw.